### DISCUSSION TITLE
외부 스크립트 연동 문의

### DISCUSSION THREAD

**Discussion by Archisketch-Cole** [commerce-api-naver/commerce-solutions#73] [묻고 답하기]
안녕하세요.
아키스케치 팀의 김준수라고합니다.

Archisketch App을 네이버 커머스 솔루션에 구축하고 있는데요.
혹시 제공해주시는 Hook 중 네이버 커머스 웹사이트 HTML에 script 엘레먼트를 붙일 수 있는 `Hook`도 있는지 궁금하여 여쭤봅니다.

구동방식은 채널톡과 유사할듯합니다.

+ 만약, 스크립트 설치가 가능하다면 유저 정보를 동기화 시킬 수 있게 제공되는 `FrontAPI`도 존재하는지 궁금합니다.
+ 혹시 스크립트 삽입 Hook이 없다면, 저희가 어떻게 대응하면 좋을지도 같이 알려주시면 감사하겠습니다. 🙇‍♂️

고맙습니다.

**참고**

- 아키스케치 x 카페24
https://docs.archisketch.com/reference/%EC%B9%B4%ED%8E%9824cafe24

- 아키스케치 x 워드프레스
https://docs.archisketch.com/reference/%EC%9B%8C%EB%93%9C%ED%94%84%EB%A0%88%EC%8A%A4wordpress

---

**Comment by commerce-api-naver** [comment]
안녕하세요. 커머스API 이민석입니다.

> [!IMPORTANT]
> 게시해주신 내용은 커머스솔루션 개발사 연동 관련 문의로 판단되어 관리자가 게시 공간을 이동하였습니다.
> - 최초 게시된 공간: 커머스API 내 스토어 애플리케이션 기술지원 공간 (`/commerce-api`)
> - 이동된 공간: 커머스솔루션 개발사 기술지원 공간 (`/commerce-solutions`)

구매자 프론트-엔드 환경 요소를 제어하는 기능은 커머스API로 제공하지 않습니다.
따라서 커머스솔루션 역시 구매자 프론트-엔드 환경에서 sciprt 요소를 추가하거나 제어하는 방식으로 외부 스크립트를 추가할 수는 없습니다.

구매자 프론트-엔드 환경에서의 HTML 구성 요소 변경은 구매 경험에도 영향을 미치기에 스마트스토어 서비스 정책으로 지원하지 않음으로 답변드립니다.
